index.html

```diff
@@ -25,54 +31,44 @@
             <p>Ayudando a los Hispanohablantes a Aprender Terminolog&iacute;a de Desarollo Web</p>
         </div>
     </header>
-    <div class="nav-placeholder" style="height: 42px;">
+<h4 class='searchlabel'>Select an element to view definition, memorization aids, audio pronunciation, and code sample:<br>Selecione un elemento para ver la definici&oacute;n, ayuda para recordar, la pronunciaci&oacute;n con audio, y ejemplos de como usar la etiqueta:</h4>
+
+<div class="sticky-combo">
         <nav class="">
             <div class="nav-inner">
                 <div class="nav-inner-most">
```

Comment:
@ahaynes-gpsw so i basically taught myself how to do this..
I basically followed along with the guy teaching how to do this sticky bar:+1: